---
preprint: false
title: eQTLs identify regulatory networks and drivers of variation in the individual response to sepsis
authors:
 - Katie L. Burnham
 - Nikhil Milind
 - Wanseon Lee
 - Andrew J. Kwok
 - Kiki Cano-Gamez
 - Yuxin Mi
 - Cyndi G. Geoghegan
 - Ping Zhang
 - GAinS Investigators
 - Stuart McKechnie
 - Nicole Soranzo
 - Charles J. Hinds
 - Julian C. Knight
 - Emma E. Davenport
journal: Cell Genomics
date: 2024-06-18
doi: 10.1016/j.xgen.2024.100587
---

Sepsis is a clinical syndrome of life-threatening organ dysfunction caused by a dysregulated response to infection, for which disease heterogeneity is a major obstacle to developing targeted treatments. We have previously identified gene-expression-based patient subgroups (sepsis response signatures \[SRS\]) informative for outcome and underlying pathophysiology. Here, we aimed to investigate the role of genetic variation in determining the host transcriptomic response and to delineate regulatory networks underlying SRS. Using genotyping and RNA-sequencing data on 638 adult sepsis patients, we report 16,049 independent expression (eQTLs) and 32 co-expression module (modQTLs) quantitative trait loci in this disease context. We identified significant interactions between SRS and genotype for 1,578 SNP-gene pairs and combined transcription factor (TF) binding site information (SNP2TFBS) and predicted regulon activity (DoRothEA) to identify candidate upstream regulators. Overall, these approaches identified putative mechanistic links between host genetic variation, cell subtypes, and the individual transcriptomic response to infection.
