---
preprint: true
title: High false sign rates in transcriptome-wide association studies
authors:
 - Peter A. Gerlach*†
 - Nikhil Milind*†
 - Jeffrey P. Spence†
 - Jonathan K. Pritchard†
journal: bioRxiv
date: 2025-12-20
doi: 10.64898/2025.12.19.695550
---

Transcriptome-wide association studies (TWAS) are widely used to identify genes involved in complex
traits and to infer the direction of gene effects on traits. However, despite their popularity, it
remains unclear how accurately TWAS recover the true direction of a gene's effect on a trait. Here,
we estimate the false sign rate (FSR) of TWAS for plasma proteins, leveraging the expectation that
increased gene expression should generally increase protein expression. We then extend this
framework to complex traits, where loss-of-function burden tests provide the expected
direction-of-effect. In both analyses, we observe high discordance with expectations, with TWAS
showing an FSR of 23% for plasma proteins and 33% for complex traits. While colocalization-based
filtering reduced the FSR, substantial discordance remained, and with substantial loss of recall.
However, when we restricted gene-direction assignments for plasma proteins to using only relevant
tissues in combination with colocalization-based filtering, the FSR dropped to 11%, and to just 5%
if we excluded brain-specific proteins. We propose that much of the sign discordance arises when
eQTLs in non-trait-relevant tissues tag GWAS-associated haplotypes via distinct, tightly-linked
regulatory variants, yielding spurious TWAS associations with the correct genes but with unreliable
direction-of-effect. These findings show that TWAS-based direction-of-effect estimates should be
interpreted with caution and raise concerns about the reliability of TWAS more broadly.
