---
name: Software and Data
---

# Software

#### CNV GDRC Paper

We use rare variant burden tests to estimate monotonicity and trait buffering. Scripts to accomplish this can be found [here](https://zenodo.org/records/13864152).

#### CHEERS Update

CHEERS was a tool written to test for enrichment of SNPs in ATAC-seq peaks specific to cell types. I rewrote this tool for Python 3 and made it more efficient. It can be found on a separate branch [here](https://github.com/TrynkaLab/CHEERS/tree/python3).

#### LeanGoShifter

[goshifter](https://github.com/immunogenomics/goshifter) was a tool written to test the enrichment of GWAS loci in genomic annotations. I rewrote the basic functionality in Python 3 to make it more efficient. It can be found [here](https://github.com/nikhilmilind/LeanGoShifter).

# Data

#### CNV GDRC Paper

We performed rare variant burden tests using loss-of-function (LoF) variants, deletions, and duplications in the UK Biobank for over 400 traits. These can be found [here](https://zenodo.org/records/13852455).

#### Alzheimer's Disease Summary Statistics

We used co-expression networks to generate latent gene expression factors, that we mapped to the genome using a single-variant association analysis approach. We also mapped Alzheimer's case subtypes using the same approach. The summary statistics from these association tests can be found [here](https://datadryad.org/stash/dataset/doi:10.5061/dryad.rbnzs7h84).
